resources/install.rb

```diff
@@ -359,10 +359,17 @@
     only_if "#{nginx_binary} -t"
   end
 
+  notify_group 'Delay nginx configuration validation' do
+    notifies :write, 'log[Validate nginx config]', :delayed
+    action :run
+  end
```

Comment:
Would it not be cleaner to just use `delayed_action` with the log resource instead?